### ISSUE TITLE
calling 'isCertainlySingularAt' by user breaks '.isProbablySmoothAt' (unknown key) 

### ISSUE THREAD

**Issue by jakobkroeker** [jakobkroeker/FiniteFieldExperiments.M2#86]
calling isCertainlySingularAt by user breaks '.isProbablySmoothAt' :

`isProbablySmoothAt()` becames unknown key.

**Comment by jakobkroeker** [comment]
cannot reproduce...

**Comment by jakobkroeker** [comment]
failing input  
```
          R = QQ[x,y]
          I = ideal(x^5-y^7);
          bbI = blackBoxIdeal I;
          smoothPoint = matrix{{8,4_QQ}}
          bbI.isProbablySmoothAt(smoothPoint)
          origin = matrix{{0,0_QQ}}
          bbI.isProbablySmoothAt(origin)
          bbI.isCertainlySingularAt(origin)
          bbI.setSingularityTestOptions(4,1)
          bbI.isCertainlySingularAt(origin)
          bbI.isProbablySmoothAt(origin)
```

**Comment by jakobkroeker** [comment]
fixed with 4e936650af8257e302005d656184add4ca41bd83
